app/code/Magento/Theme/Controller/Result/MessagePlugin.php

```diff
@@ -59,6 +60,11 @@ class MessagePlugin
      */
     protected $sessionConfig;
 
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
```

Comment:
Please make it private
```suggestion
    private $scopeConfig;
```